
string name #load or save as input 

---

bool Status #Execution Status
